### ISSUE TITLE
FNX-4929 ⁃ Animation: opening an existing tab from the home screen

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#1287]
UX designs: #167 

Opening an existing tab from the home screen - Transition duration 120ms

• Apply Android ripple overlay on release of pressed state on tab.
• After ripple animation, tab moves up and simultaneously fades out to 0% opacity.
• Browser chrome and website screen moves up and simultaneously fades in to 100% opacity.

**Comment by vesta0** [comment]
Moving this out of the sprint.

**Comment by ekager** [comment]
I implemented a similar shared transition that "opens" the tab into the browser, but it does not match the mocks directly. We can revisit this later if it needs tweaking.

**Comment by sblatz** [comment]
We are currently in the process of redesigning the home and tab screens. We are also waiting on a Google library update that will fix a crash we had with this ticket. 

@ekager can you own tracking this ticket's corresponding Google library update? :)

**Comment by sblatz** [comment]
@mozilla-mobile/ux can we make sure this animation is considered in the redesign?

**Comment by lime124** [comment]
@sblatz @vesta0  since that effort is on hold at the moment do we want to size this and see if it's something easy we can do for the release? but it also seems like we are waiting on somthing from google anyhow so maybe it's still blocked?

/cc @topotropic

**Comment by sblatz** [comment]
This animation would be blocked on a google library update too, yes.

**Comment by sblatz** [comment]
Closing this as they're now in the app.
